flink-streaming-connectors/flink-connector-twitter/pom.xml

```diff
@@ -35,6 +35,11 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<properties>
+		<hbc-core.version>2.2.0</hbc-core.version>
```

Comment:
I wanted to allow users passing a custom property using `-Dhbc-core.version=a.b.c`. I didn't change the version, just introduced the prop.